root/static/scripts/release-editor/fields.js

```diff
@@ -511,7 +515,8 @@ class Medium {
              );
     });
     this.confirmedFeatOnTrackTitles = ko.observable(
-      this.hasFeatOnTrackTitles.peek(),
+      !self.tracksUnknownToUser() &&
+      self.tracks().some(t => t.hasOrigFeatOnTitle),
```

Comment:
Shouldn't this be `hasFeatOnOrigTitle`?

Reply:
Yikes, yes! No `@flow strict` on this file. :-(

Let me try to figure out why this didn't show up when I tested manually (and rebase after #3468  is merged and try to add a test).

Reply:
Hmm, tricky. I think that even though this gets initialized to false due to this typo, `tracksLoaded` later sets it to true by calling `this.confirmedFeatOnTrackTitles(this.hasFeatOnTrackTitles())`. I'm not sure if a test could catch this (but I'm still happy to add one after #3468 is in).

(Side note: I didn't want to change it here, but "hasFeat**In**Title" sounds a bit more grammatically correct than "hasFeat**On**Title". :-))

Reply:
If @mwiencek agrees, then feel free to change it in a second commit once this is working :)

Reply:
Okay. :-)

Sorry if it wasn't clear, but this change is working now, as far as I know.

Reply:
> (Side note: I didn't want to change it here, but "hasFeatInTitle" sounds a bit more grammatically correct than "hasFeatOnTitle". :-))

That sounds better to me too. 👍